### PR TITLE
v1.0 backports 18-09-28

### PR DESCRIPTION
**Pull request by jrfastab**
PR:
https://github.com/cilium/cilium/pull/5708

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5725)

### CODE REVIEW

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

**Comment by jrfastab** [comment]
test-me-please

File: daemon/daemon.go [hunk 1]
```diff
@@ -926,14 +926,13 @@ func (d *Daemon) syncLXCMap() error {
 		if added {
 			log.WithField(logfields.IPAddr, ip).Debugf("Adding local ip to endpoint map")
 		}
+		delete(existingEndpoints, ip.String())
```
Comment (ianvernon) [inline daemon/daemon.go:929]: Sign off on your commit. Elaborate on "It seems we are losing endpoint IPs" by providing evidence that backs up your claim.

**Review by ianvernon — CHANGES_REQUESTED** [review]

File: daemon/daemon.go (hunk 1 above)
Comment (jrfastab) [reply daemon/daemon.go:929]: its a test patch to track down a failure.

Comment (ianvernon) [reply daemon/daemon.go:929]: Thanks for clarifying. I will hold off on reviewing in that case.

Comment (jrfastab) [reply daemon/daemon.go:929]: for what its worth I generally don't sign-off test patches because it stops folks from committing them accidentally.

**Comment by jrfastab** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

File: daemon/daemon.go [hunk 2]
```diff
@@ -926,14 +926,13 @@ func (d *Daemon) syncLXCMap() error {
 		if added {
```
Comment (ianvernon) [inline daemon/daemon.go:926]: typo in commit: endpoing --> endpoint (you can amend the commit and re-push, but don't have to run tests again for this to be merged)

**Review by ianvernon — APPROVED** [review]

**Review by raybejjani — APPROVED** [review]

**Pull request merged**